src/main/java/com/bandwidth/sqs/action/SetQueueAttributesAction.java

```diff
@@ -7,17 +7,17 @@
 import com.amazonaws.services.sqs.model.transform.SetQueueAttributesRequestMarshaller;
 import com.amazonaws.services.sqs.model.transform.SetQueueAttributesResultStaxUnmarshaller;
 import com.bandwidth.sqs.action.adapter.SqsAwsSdkAction;
-import com.bandwidth.sqs.queue.SqsQueueAttributes;
+import com.bandwidth.sqs.queue.SqsQueueAttributeChanges;
```

Comment:
k, then everywhere it says "attributes" it must now say "attributeChanges" or "changes". Param names, var names, method names, field names etc...